### PR TITLE
Add support for Plone 5.1.x

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

File: collective/deletepermission/testing.py [hunk 1]
```diff
@@ -6,9 +6,13 @@
 from plone.app.testing import PLONE_FIXTURE
 from plone.app.testing import PloneSandboxLayer
 from plone.app.testing import setRoles, TEST_USER_ID, TEST_USER_NAME, login
+from pkg_resources import get_distribution
 import collective.deletepermission.tests.builders
 
 
+IS_PLONE_5_OR_GREATER = get_distribution('Plone').version >= '5'
```
Comment (jone) [inline collective/deletepermission/testing.py]: In testing, I'd rather use `ftw.testing.IS_PLONE_5`.

File: collective/deletepermission/tests/base.py [hunk 2]
```diff
@@ -27,13 +27,11 @@ def set_local_roles(self, context, user, *roles):
...
 
-    def get_status_messages(self):
-        request = self.layer['request']
-        messages = [msg.message for msg in IStatusMessage(request).show()]
-        return messages
-
     def get_actions(self):
-        return browser.css('#plone-contentmenu-actions .actionMenuContent a').text
+        if testing.IS_PLONE_5_OR_GREATER:
+            return browser.css('#plone-contentmenu-actions .plonetoolbar-content-action a').text
```
Comment (jone) [inline collective/deletepermission/tests/base.py]: We could use the `editbar` page object here -> https://github.com/4teamwork/ftw.testbrowser/blob/master/ftw/testbrowser/pages/editbar.py#L79

File: development.cfg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [buildout]
 extends =
-    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-4.3.x.cfg
+    test-plone-4.3.x.cfg
     https://raw.github.com/4teamwork/ftw-buildouts/master/plone-development.cfg
 
 package-name = collective.deletepermission
```
Comment (jone) [inline development.cfg]: ✂️ can now be removed.

File: collective/deletepermission/testing.py (hunk 1 above)
Comment (mbaechtold) [reply collective/deletepermission/testing.py]: We don't have `ftw.testing` installed. Should I install it only for this "constant"?

File: collective/deletepermission/tests/base.py (hunk 2 above)
Comment (mbaechtold) [reply collective/deletepermission/tests/base.py]: Oh yes, nice! Updated...

File: development.cfg (hunk 3 above)
Comment (mbaechtold) [reply development.cfg]: Removed!

**Pull request merged**